### DISCUSSION TITLE
Should we change the repository/website name?

### DISCUSSION THREAD

**Discussion by sarmadhamdani02** [sarmadhamdani02/WebGames#16] [Q&A]
What do you think? should we change the name "Web Games" to something more fancy? 🤔 
If you are having some names please comment below 👇🙂

**Comment by avinashji1302** [comment]
Multiple Web Games

cause it has multiple games

**Reply by sarmadhamdani02** [reply]
Hi @avinashji1302 👋

It makes sense 👏 but I think it can be much better. 

I think "Multiple Games" conveys its meaning but it will not be much appealing and eye-catchy for other contributors and uses 🤔
